src/Common/ErrorCodes.cpp

```diff
@@ -622,6 +622,7 @@
     M(741, TABLE_UUID_MISMATCH) \
     M(742, DELTA_KERNEL_ERROR) \
     M(743, ICEBERG_SPECIFICATION_VIOLATION) \
+    M(744, SESSION_ID_EMPTY) \
```

Comment:
We could use `BAD_ARGUMENTS` to avoid adding too specific error codes that we use only once, but a separate error code is also fine